swift/ql/lib/codeql/swift/elements/MacroRole.qll

```diff
@@ -0,0 +1,35 @@
+/**
+ * This module provides a hand-modifiable wrapper around the generated class `MacroRole`.
+ */
+
+private import codeql.swift.generated.MacroRole
+
+class MacroRole extends Generated::MacroRole {
```

Comment:
Could you add some QLDoc to describe what a `MacroRole` is? 🙏

It looks like you added it to the autogenerated class via the `schena.py` file (which is awesome!), but I think we should port it to the user-facing file as well.